hack/debug-network.sh

```diff
@@ -0,0 +1,574 @@
+#!/bin/bash
```

Comment:
If you have library functions to register, prefer to put them in files under `hack/lib`